cluster-provision/k8s/1.31/manifests/prometheus/alertmanager/alertmanager-alertmanager.yaml

```diff
@@ -2,34 +2,36 @@ apiVersion: monitoring.coreos.com/v1
 kind: Alertmanager
 metadata:
   labels:
-    alertmanager: main
     app.kubernetes.io/component: alert-router
+    app.kubernetes.io/instance: main
     app.kubernetes.io/name: alertmanager
     app.kubernetes.io/part-of: kube-prometheus
-    app.kubernetes.io/version: 0.21.0
+    app.kubernetes.io/version: 0.27.0
   name: main
   namespace: monitoring
 spec:
-  image: quay.io/prometheus/alertmanager:v0.21.0
+  image: quay.io/prometheus/alertmanager:v0.27.0
   nodeSelector:
     kubernetes.io/os: linux
   podMetadata:
     labels:
       app.kubernetes.io/component: alert-router
+      app.kubernetes.io/instance: main
       app.kubernetes.io/name: alertmanager
       app.kubernetes.io/part-of: kube-prometheus
-      app.kubernetes.io/version: 0.21.0
-  replicas: 1
+      app.kubernetes.io/version: 0.27.0
+  replicas: 3
```

Comment:
We won't need three replicas in our test clusters - its better to keep this at 1.

Reply:
@brianmcarey ok then i will modify in all folder . Thankyou

Reply:
@brianmcarey i have modified to all the manifest to the recent version. Please check .